src/biip/gs1/_messages.py

```diff
@@ -78,6 +84,29 @@ def parse(
 
         return cls(value=value, element_strings=element_strings)
 
+    @classmethod
+    def parse_hri_string(value: str) -> GS1Message:
+        """Parse the GS1 string given in HRI (human readable interpretation) format.
+
+        Args:
+            value: The GS1 string to parse.
+
+        Returns:
+            Data class created by ``parse``.
```

Comment:
Just say the same thing as `parse()` does. That this method uses `parse()` internally is an implementation detail.

```suggestion
            A message object with one or more element strings.
```